pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -120,6 +166,27 @@ def __init__(
 
 
 class ConvSequence(SequentialWithOutChannels):
+    r"""Sequence of convolutional blocks that occurs repeatedly in :cite:`ULVL2016`.
+
+    Each sequence contains three
+    :class:`~pystiche_paper.ulyanov_et_al_2016._modules.ConvBlock` blocks while the
+    first two have a ``kernel_size of 3`` and the last one a ``kernel_size of 1``.
```

Comment:
The "``<text>``" are literal blocks commonly used to display code. "kernel_size of 3" is not valid python code. 

```suggestion
    first two uses ``kernel_size == 3``, the last one uses ``kernel_size == 1``.
```